code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -55,12 +93,25 @@
 	You have seen lights in the distance... they foreshadow the arrival of outsiders to your domain. \
 	Ensure your nest remains protected at all costs."
 	assignedrole = "Ash Walker"
+	var/datum/team/ashwalkers/team
+	var/obj/structure/ash_walker_eggshell/eggshell
+
+/obj/effect/mob_spawn/human/ash_walker/Destroy()
+	eggshell = null
+	return ..()
+
+/obj/effect/mob_spawn/human/ash_walker/allow_spawn(mob/user, silent = FALSE)
```

Comment:
I disagree with limiting ashwalker spawns to one per player (unless they revive another ashwalker and get lucky) as ashwalker is often a dangerous and lone role, with there not really being any reason to deny it.

Reply:
also remember that you start with 2 bodies in the nest. you can also be revived by being dragged to the tentacle. its meant to be a team thing, and the focus is being, get to the tentacle and you come back to full health. i think the having it limited in respawns balances that

Reply:
we kinda have to take into consideration when there's not enough players for the team thing to work out; there's a lot of solo ashwalker play here